content/systems/filecoin_mining/sector/sector-faults.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Sector Faults
+weight: 4
+dashboardWeight: 2
+dashboardState: stable
+dashboardAudit: wip
+dashboardTests: 0
+---
+
+# Sector Faults
+
+It is very important for storage providers to have a strong incentive to both report the failure to the chain and attempt recovery from the fault in order to uphold the storage guarantee for the networkʼs clients. Without this incentive, it is impossible to distinguish an honest minerʼs hardware failure from malicious behavior, which is necessary to treat miners fairly. The size of the fault fees depend on the severity of the failure and the rewards that the miner is expected to earn from the sector to make sure incentives are aligned. The three types of sector storage fault fees are:
+
+* **Sector fault fee:** This fee is paid per sector per day while the sector is in a faulty state. The size of the fee is slightly more than the amount the sector is expected to earn per day in block rewards. If a sector remains faulty for more than 2 consecutive weeks, the sector will pay a termination fee and be _removed from the chain state_. The initial value for this fee is 2.14 days worth of block reward for the faulty sector. As storage miner reliability increases above a reasonable threshold, the risk posed by these fees decreases rapidly.
+* **Sector fault detection fee:** This is a one-time fee paid in the event of a failure if the miner does not report it honestly and instead the unreported failure is caught by the   chain. Given the probabilistic nature of our PoSt checks, this will be 5 daysʼ worth of block reward that would be expected to be earned by a particular sector.
```

Comment:
this number has been revised to 3.5 but I expect some more updates to the parameter.

Reply:
It's either in [monies.go](https://github.com/filecoin-project/specs-actors/blob/788f3e0d138a87d5e84b1740d1ee67345ab636b6/actors/builtin/miner/monies.go#L40) or different policy.go files